recipes/pyferret/site_specific.patch

```diff
@@ -0,0 +1,199 @@
+diff -Naur PyFerret-67aed0fa29f2d3f28e624030d746d23d146fc387.orig/external_functions/ef_utility/platform_specific.mk.x86_64-linux PyFerret-67aed0fa29f2d3f28e624030d746d23d146fc387/external_functions/ef_utility/platform_specific.mk.x86_64-linux
+--- external_functions/ef_utility/platform_specific.mk.x86_64-linux	2016-06-29 20:09:51.000000000 -0300
++++ external_functions/ef_utility/platform_specific.mk.x86_64-linux	2016-07-06 14:40:59.260485464 -0300
+@@ -23,8 +23,7 @@
+ LD		= gfortran
+ LD_DYN_FLAGS	= -fPIC -m64 -shared -Wl,--no-undefined
+ SYSLIBS		= ../ef_utility/copy_ferret_ef_mem_subsc.o \
+-		  -L$(INSTALL_FER_DIR)/lib64/$(PYTHON_EXE)/site-packages/pyferret \
+-		  -L$(INSTALL_FER_DIR)/lib/$(PYTHON_EXE)/site-packages/pyferret \
++		  -L$(SP_DIR)/pyferret \
```

Comment:
Note my "hack" here to get the external functions build to find the `libpyferret.so`. (`SP_DIR` expands to python site-packages directory.)